pkg/controller/machine/controller.go

```diff
@@ -234,25 +289,57 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, nil
 	}
 
-	exist, err := r.actuator.Exists(ctx, cluster, m)
+	instanceExist, err := r.actuator.Exists(ctx, cluster, m)
```

Comment:
This is 100% a tiny nit, but this would normally be `instanceExists` with an s like the method that's called. It reads a little weird without it.